.travis.yml

```diff
@@ -22,8 +22,8 @@ env:
   matrix:
     - ROS_DISTRO="kinetic"
     - ROS_DISTRO="kinetic" ROS_REPO=ros-shadow-fixed
-    - ROS_DISTRO="melodic"
-    - ROS_DISTRO="melodic" ROS_REPO=ros-shadow-fixed
+    - ROS_DISTRO="melodic" UPSTREAM_WORKSPACE='github:ros-industrial/fanuc#indigo-devel'
+    - ROS_DISTRO="melodic" UPSTREAM_WORKSPACE='github:ros-industrial/fanuc#indigo-devel' ROS_REPO=ros-shadow-fixed
```

Comment:
@ipa-mdl: is `UPSTREAM_WORKSPACE` like this only supported in `master` of `industrial_ci`?

Reply:
Yes.
The legacy version only supports .rosinstall URLs/files.

Reply:
Hm. 1-0 for `master` then.

I'll see if I'll switch.